components/user_settings/security/user_settings_security.jsx

```diff
@@ -93,12 +94,13 @@ export default class SecurityTab extends React.Component {
 
     componentDidMount() {
         if (this.props.enableOAuthServiceProvider) {
-            getAuthorizedApps(
-                (authorizedApps) => {
-                    this.setState({authorizedApps, serverError: null}); //eslint-disable-line react/no-did-mount-set-state
-                },
-                (err) => {
-                    this.setState({serverError: err.message}); //eslint-disable-line react/no-did-mount-set-state
+            this.props.actions.getAuthorizedApps().then(
+                ({data, error}) => {
+                    if (data) {
+                        this.setState({authorizedApps: data, serverError: null}); //eslint-disable-line react/no-did-mount-set-state
```

Comment:
As I mentioned in https://github.com/mattermost/mattermost-redux/pull/776, it might be best to keep this data in redux, but I don't feel very strongly about that. Just thought I'd mention it here as well since different people are reviewing to this PR and they may have something to add to that conversation.